File: app/src/main/java/com/santalu/myapplication/MainActivity.java
package com.santalu.myapplication;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.Button;
import android.widget.TextView;
import com.santalu.maskedittext.MaskEditText;

public class MainActivity extends AppCompatActivity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
    MaskEditText etCreditCart = findViewById(R.id.et_credit_cart);
    MaskEditText etPhoneNumber = findViewById(R.id.et_phone_number);
    MaskEditText etDate = findViewById(R.id.et_date);
    TextView tvCreditCart = findViewById(R.id.tv_credit_cart);
    TextView tvPhoneNumber = findViewById(R.id.tv_phone_number);
    TextView tvDate = findViewById(R.id.tv_date);
    Button button = findViewById(R.id.button);

    button.setOnClickListener(view -> {
      tvCreditCart.setText(String.format("%s %s", getString(R.string.hint_credit_cart_number),
          etCreditCart.getRawText()));
      tvPhoneNumber.setText(String.format("%s %s", getString(R.string.hint_phone_number),
          etPhoneNumber.getRawText()));
      tvDate.setText(String.format("%s %s", getString(R.string.hint_date), etDate.getRawText()));
    });
  }
}

File: app/src/main/java/com/santalu/myapplication/MainActivity.kt
package com.santalu.myapplication

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.etCreditCart
import kotlinx.android.synthetic.main.activity_main.etDate
import kotlinx.android.synthetic.main.activity_main.etPhoneNumber
import kotlinx.android.synthetic.main.activity_main.show
import kotlinx.android.synthetic.main.activity_main.tvCreditCart
import kotlinx.android.synthetic.main.activity_main.tvDate
import kotlinx.android.synthetic.main.activity_main.tvPhoneNumber

class MainActivity : AppCompatActivity() {

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_main)

    show.setOnClickListener {
      tvCreditCart.text = etCreditCart.text
      tvPhoneNumber.text = etPhoneNumber.text
      tvDate.text = etDate.text
    }
  }
}
